ew/cmd/mutation/mutationcmds.py

```diff
@@ -407,6 +407,8 @@ async def piss(cmd):
                     return await fe_utils.send_message(cmd.client, cmd.message.channel, fe_utils.formatMessage(cmd.message.author, response))
 
                 response = "You piss HARD and FAST right onto {}!!".format(target_member.display_name)
+            elif ewcfg.mutation_id_quantumlegs in mutations:
```

Comment:
dude what if you made it so quantum legs spent its charge if you quantum pissed

Reply:
You're right. I let my biases get in the way of game balance. I'll add the cooldown